addons/project/models/project.py

```diff
@@ -216,7 +216,7 @@ def _get_default_rating_status(self):
     date = fields.Date(string='Expiration Date', index=True, tracking=True)
     subtask_project_id = fields.Many2one('project.project', string='Sub-task Project', ondelete="restrict",
         help="Project in which sub-tasks of the current project will be created. It can be the current project itself.")
-    allow_subtasks = fields.Boolean('Sub-tasks', compute='_compute_allow_subtasks')
+    allow_subtasks = fields.Boolean('Sub-tasks', compute='_compute_allow_subtasks', store=True, readonly=False)
```

Comment:
This field will never be recomputed, as there is no trigger. So the stored value could be wrong.

Reply:
@tivisse The compute method _compute_allow_subtasks is already overrided in the industry_fsm and over there the dependency is added for the "is_fsm".

Thanks

Reply:
Yes, but it means that if you only install project, this is wrong ;)

Reply:
@tivisse , Right 👍 , I have done the changes accordingly. Thanks